app/views/units/index.html.erb

```diff
@@ -1,7 +1,23 @@
 <h1>Units</h1>
 
-<ul>
-  <% @units.each do |unit| %>
-    <li><%= unit.name %></li>
-  <% end %>
-</ul>
+<div class="container">
+<table class="table table-hover table-condensed">
+    <thead>
+      <tr>
+        <th class="col-sm-6">Unit Name</th>
```

Comment:
Looks like you're using Bootstrap classes here, but don't believe that you've imported the necessary Bootstrap files to make this work. I imagine you were looking at an example for how to create a table that included Bootstrap? Recommend looking online for HTML specific resources instead, (e.g. if you search for HTML table, generally the W3Schools links are pretty good: [HTML table](http://www.w3schools.com/html/html_tables.asp)).